### ISSUE TITLE
wrong return type

### ISSUE THREAD

**Issue by sjanssen2** [flatironinstitute/deepblast#93]
Hi Jamie,
I try to check the traditional hardmax NW but code execution fails unless I remove the `, _` on the left hand side. I think the [torch.max function](https://pytorch.org/docs/stable/generated/torch.max.html) only returns a single Tensor, doesn't it?
https://github.com/flatironinstitute/deepblast/blob/ab48e4f08091e26b0b5e6d3099700eb0c4b99538/deepblast/ops.py#L7

**Comment by mortonjt** [comment]
Yes, you are right -- this looks like it was inherited from an older version of pytorch where max returned both the value and indices.  This will need to updated.
